src/test/javascript/spec/component/modeling-exercise/modeling-exercise-update.component.spec.ts

```diff
@@ -266,18 +275,4 @@ describe('ModelingExercise Management Update Component', () => {
         comp.getCheckedFlag(false);
         expect(comp.checkedFlag).toBe(false);
     });
-
-    it('should delete example submission', fakeAsync(() => {
-        comp.modelingExercise = new ModelingExercise(UMLDiagramType.Flowchart, undefined, undefined);
-        const exampleSubmission1 = new ExampleSubmission();
-        exampleSubmission1.id = 1;
-        const exampleSubmission2 = new ExampleSubmission();
-        exampleSubmission2.id = 2;
-        comp.modelingExercise.exampleSubmissions = [exampleSubmission1, exampleSubmission2];
-        spyOn(exampleSubmissionService, 'delete').and.returnValue(of(new HttpResponse({})));
-        comp.deleteExampleSubmission(2, 1);
-        tick();
-        expect(exampleSubmissionService.delete).toHaveBeenCalledWith(2);
```

Comment:
This deletes the last usage of `exampleSubmissionService`, so the variable (and import) itself should also be removed I suppose?